### DISCUSSION TITLE
Unable to open AU plugin on M1

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#885] [Help]
When using AU component compiled for Intel on a M1 Mac, first loading is working.

Anytime you reopen the GUI in AG there is an AU bridge error stating plugin is loaded in an unsupported way.

Not sure If it happens to on Intel Mac as normally I dont use AU (but AU bridge is usefull on M1 Mac).

Happens for a lot of plugins I tried, especially Izotope Ozone products.
![Screen Shot 2022-01-15 at 15 58 46](https://user-images.githubusercontent.com/56123216/149626463-32db7d4b-2dd7-4118-b3d7-ddb8a74c1b5d.png)

**Comment by emack999** [comment]
It'll be quite some time before all the plugins you use are compatible with M1 without Rosetta.
There's really no point to using an M1 for pro audio right now.  Intel Mac is still far more stable and more than enough.

**Comment by Henri75** [comment]
I prefer the intel of course, but right now my M1X running native DAW is 3x
faster than my i9 ...
I use izotope plugin daily on cubase using rosetta and S1, so the bridge is
working great on M1.
There is something specific to Juce or AG that broke compatibility....
